project/build.sbt

```diff
@@ -1,7 +1,4 @@
 // Used by VersionUtil to get gitHash and commitDate
 libraryDependencies += "org.eclipse.jgit" % "org.eclipse.jgit" % "4.11.0.201803080745-r"
 
-
-Compile / unmanagedSourceDirectories +=
-  baseDirectory.value / "../language-server/src/dotty/tools/languageserver/config"
```

Comment:
Did you mean to remove this?

Reply:
I think this change comes from @tgodzik's commit.